GSA/digitalgov.gov
# Remove topics pages with zero items

## Summary

This PR will remove topics pages that have been listed with zero items 

### Preview

[Link to Preview](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/cm-topics-removal/)



### Solution

Deleted folders that were listed in the spreadsheet with zero items.

### How To Test

Add the following paths to the preview link and ensure they are giving a 404 error 

1. `topics/cops-office`
2. `topics/defense-intelligence-agency`
3. `topics/dgu`
4. `topics/dia`
5. `topics/digitalgov-university`
6. `topics/first-responder-network-authority`
7. `topics/first-responders-network`
8. `topics/national-museum-of-african-american-history-and-culture`
9. `topics/nmaahc`
10. `topics/office-of-community-oriented-policing-services`
11. `topics/office-of-inspector-general`
12. `topics/oig`
13. `topics/us`
14. `topics/usfs`
15. `topics/whats-new`
16. `topics/work-in-progress`

---

### Dev Checklist
